playground/backend/containers/go/Dockerfile

```diff
@@ -58,17 +58,6 @@ ENV PROPERTY_PATH=/opt/playground/backend/properties.yaml
 ## Copy build result
 COPY src/configs /opt/playground/backend/configs/
 
-# Install mitmpoxy
-RUN mkdir /opt/mitmproxy &&\
-    cd /opt/mitmproxy &&\
-    wget -q https://snapshots.mitmproxy.org/7.0.4/mitmproxy-7.0.4-linux.tar.gz &&\
-    tar -zxvf mitmproxy-7.0.4-linux.tar.gz &&\
-    mkdir /usr/local/share/ca-certificates/extra
-COPY allow_list_proxy.py /opt/mitmproxy/
-COPY allow_list.py /opt/mitmproxy/
-ENV HTTP_PROXY="http://127.0.0.1:8081"
-ENV HTTPS_PROXY="http://127.0.0.1:8081"
-
 COPY src/properties.yaml /opt/playground/backend/properties.yaml
```

Comment:
Again, why not rely on a ConfigMap and mount that to the container, decoupling configuration from binary?  If we are still stuck with appengine, terraform has a way to provide environment variables via the app_engine_flexible_app_version block https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/app_engine_flexible_app_version#env_variables

Reply:
This file contains the settings which would not be changed after GKE deployment